scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -131,68 +128,214 @@ def make_create_workspace_request(workspace_name, auth_domains, namespace=NAMESP
     return create_ws_request
 
 
+def copy_workspaces_workflows(destination_workspace_namespace, destination_workspace_name, source_workspace_namespace, source_workspace_name):
+    """Copy Van Allen Lab workspaces Workflows."""
+    # Get the list of all the workflows
+    try:
+        workflow_list = fapi.list_workspace_configs(source_workspace_namespace, source_workspace_name)
+
+        # Store all the workflow names
+        workflow_name_list = []
+
+        for workflow in workflow_list.json():
+            # Get workflow config (overview config)
+            workflow_config = workflow['methodRepoMethod']
+
+            workflow_name_list.append(workflow_config['methodName'])
+
+            # Get workspace config (Detailed config with inputs, oututs, etc)
+            workspace_config = fapi.get_workspace_config(source_workspace_namespace, source_workspace_name, workflow_config['methodNamespace'], workflow_config['methodName'])
+
+            # Create a workflow based on Detailed config
+            fapi.create_workspace_config(destination_workspace_namespace, destination_workspace_name, workspace_config.json())
+            print(f"Copied {workflow_config['methodName']} workflow : over to {destination_workspace_namespace}/{destination_workspace_name}")
+
+        # Check if workflows match
+        new_workflow_list = fapi.list_workspace_configs(destination_workspace_namespace, destination_workspace_name).json()
+        if workflow_list != workflow_list:
+            print(f"Error: Data Tables don't match")
+            return False, f"Error: Data Tables don't match", None
+
+    except Exception as error:
+        return False, f"Error: {error}", None
+
+    return True, None, workflow_name_list
+
+
+def find_and_replace(attr, value, replace_this, with_this):
+    """Replace an attribute in a data table row."""
+    updated_attr = None
+    if isinstance(value, str):  # if value is just a string
+        if replace_this in value:
+            new_value = value.replace(replace_this, with_this)
+            updated_attr = fapi._attr_set(attr, new_value)
+    elif isinstance(value, dict):
+        if replace_this in str(value):
+            value_str = str(value)
+            value_str_new = value_str.replace(replace_this, with_this)
+            value_new = ast.literal_eval(value_str_new)
+            updated_attr = fapi._attr_set(attr, value_new)
+    elif isinstance(value, (bool, int, float, complex)):
+        pass
+    elif value is None:
+        pass
+    else:  # some other type, hopefully this doesn't exist
+        print('unknown type of attribute')
+        print('attr: ' + attr)
+        print('value: ' + str(value))
+
+    return updated_attr
+
+
+def update_entities(workspace_name, workspace_project, replace_this, with_this):
+    """Update Data Table"""
```

Comment:
Leaving more detail would be great! "Update data table with new paths." or something like that